### PR TITLE
Create hacktoberfestFlip.css

### PR DESCRIPTION
**Pull request by saclig202**

### CODE REVIEW

File: animations/hacktoberfestFlip.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    name: 'hacktoberfestFlip',
+    author: 'saclig202'
+}
```
Comment (NiallEccles) [inline animations/hacktoberfestFlip.css:4]: This needs to go in `animations.js`

File: animations/hacktoberfestFlip.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```
Comment (NiallEccles) [inline animations/hacktoberfestFlip.css:10]: Can you remove these styles?

File: animations/hacktoberfestFlip.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+body {
+  background-color: #111;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
```
Comment (NiallEccles) [inline animations/hacktoberfestFlip.css:18]: It's best to not have your styles on the body as they conflict with other users animations. Could you remove or integrate into your CSS class?

File: animations/hacktoberfestFlip.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+.hacktoberfestFlip {
+  position: relative;
+}
+.hacktoberfestFlip span {
```
Comment (NiallEccles) [inline animations/hacktoberfestFlip.css:22]: You don't need the `span` declaration here.

File: animations/hacktoberfestFlip.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  position: relative;
+  display: inline-block;
+  font-size: 40px;
+  color: #fff;
+  text-transform: uppercase;
+  animation: flip 2.5s infinite;
+  animation-delay: calc(.2s * var(--i))
```
Comment (NiallEccles) [inline animations/hacktoberfestFlip.css:29]: This CSS variable isn't declared anywhere. Could you update it to just be a regular time?

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for submitting this. I have requested a few changes for you to make before I can accept.

**Review by saclig202 — COMMENTED** [review]
I've made the requested changes.  If you need further changes to approve the pull request, I'll be happy to make them.  Thanks!

**Pull request closed without merging**